.eslintrc.json

```diff
@@ -1,11 +1,12 @@
 {
-  "parser": "babel-eslint",
+  "parser": "@babel/eslint-parser",
   "extends": [
     "@skyscanner/eslint-config-skyscanner",
-    "plugin:flowtype/recommended"
+    "plugin:ft-flow/recommended"
   ],
   "plugins": [
-    "flowtype"
+    "ft-flow",
+    "eslint-plugin-prettier"
```

Comment:
Dependency missing?

```
ESLint couldn't find the plugin "eslint-plugin-prettier".

(The package "eslint-plugin-prettier" was not found when loaded as a Node module from the directory "/Users/matthewr/Code/backpack-foundations".)
````

Reply:
It should come from @skyscanner/eslint-config-skyscanner that is installed as a dep